sendcloud/sendcloud.php

```diff
@@ -56,7 +56,7 @@ public function __construct()
         $this->boostrap = true;
         $this->name = 'sendcloud';
         $this->tab = 'shipping_logistics';
-        $this->version = '1.4.1';
+        $this->version = '1.4.2';
         $this->author = 'Sendcloud';
         $this->author_uri = 'https://sendcloud.eu';
```

Comment:
**Issue:** this URL is currently resulting in 404; shall we take the opportunity and use `https://sendcloud.com` instead?

Reply:
Makes sense. Done